style-guide/readme.md

```diff
@@ -0,0 +1,47 @@
+> **Goal of this guide:** to help you write to fit a specific context, purpose, or audience.
+
+# How to refine your writing style
+
+## Recognise what style is and what it's building blocks are
```

Comment:
Two headings on top of each other, it feels strange to me. In my opinion every heading needs content or else why use a heading? Does this Guide needs an intro? Maybe just skip the h2 because its content is a suitable introduction already.